01-shapefile-attributes.Rmd

```diff
@@ -254,39 +239,44 @@ footpath_HARV<-lines_HARV[lines_HARV$TYPE == "footpath",]
 footpath_HARV
 #how many features in our new object
 length(footpath_HARV)
+```
+
+Our subsetting operation reduces the `features` count from 13 to 2. This means
+that only two lines have the attribute "TYPE=footpath".
 
+We can also plot our subsetted shapefiles.
+```{r plot-subset-shapefile}
 #plot just footpaths
 plot(footpath_HARV,
+     col=lines_HARV$OBJECTID, #color set by "OBJECTID" attribute
```

Comment:
how about you show me this on monday? i think i understand what you are saying - what appears as one continuous line is actually 2 feature objects (2 line objects). So a person working on the lesson might get confused because they expect to see two lines. 

where are you suggesting that we move it? if it goes after plot attribute by color then the subset part breaks up the flow of learning all of the plotting techniques at one time. so it's a bit of a chicken / egg dilemma potentially in terms of presenting like information together. hmmmm...